Engine_Revit_UI/Query/Profiles.cs

```diff
@@ -125,12 +125,12 @@ public static List<PolyCurve> Profiles(this HostObject hostObject, PullSettings
                 //{
                 //    //double aElevation = aLevel.Elevation;
                 //    //if (pullSettings.ConvertUnits)
-                //    //    aElevation = UnitUtils.ConvertFromInternalUnits(aElevation, DisplayUnitType.DUT_METERS);
+                //    //    aElevation = aElevation.ToSI(UnitType.UnitType.UT_Length);
 
                 //    //Parameter aParameter = hostObject.get_Parameter(BuiltInParameter.WALL_TOP_OFFSET);
                 //    //if (aParameter != null)
                 //    //{
-                //    //    double aOffset = UnitUtils.ConvertFromInternalUnits(aParameter.AsDouble(), DisplayUnitType.DUT_METERS);
+                //    //    double aOffset = aParameter.AsDouble().ToSI(UnitType.UnitType.UT_Length);
```

Comment:
If not needed let's remove

Reply:
I am planning to look into this method in 20Q1 - I would keep the commented code until then.